content/en/blog/posts/2024-03-27-build-release-v0.13.0.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Shipwright v0.13.0 Is Here"
+date: 2024-03-27T12:00:01-04:00
+draft: false
+author: "Enrique Encalada ([@qu1queee](https://github.com/qu1queee))"
+---
+
+After months of diligent work, just in time for [cdCon 2024](https://cd.foundation/event/cdcon-2024/), we are releasing our `v0.13.0` release. This significant milestone incorporates a bunch of enhancements, features and bug fixes. Here are the key highlights:
+
+## Action Required
+
+> After upgrading from v0.12.0 to v0.13.0, you can run the following two commands to remove unnecessary permissions on the
+  shipwright-build-webhook: `kubectl delete crb shipwright-build-webhook && kubectl delete cr shipwright-build-webhook`
+
+> We switched our CRDs storage version to `v1beta1`. We strongly advise users to migrate to our `v1beta1` API, as we intend to
+  deprecate `v1alpha1` in a future release. Note that the `v1alpha1` still served for all CRD's.
+
+## Features
+
+### Build Output Timestamp
+
+SHIP [0037](https://github.com/shipwright-io/community/blob/main/ships/0037-build-output-timestamp.md) is now implemented. Users can make use of the Build `.spec.output.timestamp` to explicitly set the resulting container image timestamp.
+
+### Tekton v1 API Adoption
+
+Our Shipwright controllers now use the Tekton v1 API when working with `TaskRuns`. See [PR](https://github.com/shipwright-io/build/pull/1435).
+
+## API Changes
+
+### Switch Storage Version to v1beta1
+
+The storage version for all CRD's has been switched to `v1beta1`. Concurrently, our Shipwright controllers have been updated to use this updated API version. The `v1alpha1` version continues to be supported.
+
+### Beta API Changes
+
+We've implemented minor adjustments to the existing `v1beta1` API for the sake of consistency, incorporating feedback gathered from the `v0.12.0` release. These are:
+
+- Renaming certain Go Types, such as `build.Spec.Source.GitSource` to `build.Spec.Source.Git`, aligning them with their respective JSON tags.
+- Designating `build.Spec.Source.Type` and `buildRun.Spec.Source.Type` as mandatory, addressing usability concerns raised by users.
+- We made `build.Spec.Source` optional. Enabling the definition of a Build without any source. This feature proves valuable
+  particularly when executing a Build only with local source.
+- Requiring `build.spec.source.git.url` upon the specification of `build.spec.source.git`.
+
+Further details can be found in [PR 1504](https://github.com/shipwright-io/build/pull/1504) and [PR 1441](https://github.com/shipwright-io/build/pull/1441).
+
+## Bugs
+
+### Conversion Webhook
+
+As the usage of the webhook increased over the last months, we've made enhancements to address certain gaps:
+
+- Previously, there was an issue with the proper conversion of BuildRuns from v1alpha1 to v1beta1 when a generated service account was utilized. This has been resolved.
```

Comment:
```suggestion
- Previously, there was an issue with the proper conversion of `BuildRun` instances from v1alpha1 to v1beta1 when a generated service account was utilized. This has been resolved.
```
Generally, put object names between backticks and add a label like "instances" or whatever accurately describes them. Make the label singular or plural, as needed. Don't make object names plural unless they are also plural in your code.

Reply:
great advice, highly appreciated!